chainer/functions/connection/convolution_2d.py

```diff
@@ -92,18 +92,12 @@ def forward_gpu(self, inputs):
                 self.bias_desc = cudnn.create_tensor_descriptor(
                     b[None, :, None, None])
 
-            self.max_workspace_size = max(
-                out_c * c * 64,  # for winograd
-                c * kh * kw * 4)
+            workspace_size = cuda.get_max_workspace_size()
```

Comment:
Do we need to check if `workspace_size >= libcudnn.getConvolutionForwardWorkspaceSize (...)` ?